packages/vsx-registry/src/browser/vsx-extensions-widget.tsx

```diff
@@ -153,4 +153,13 @@ export class VSXExtensionsWidget extends SourceTreeWidget implements BadgeWidget
         }
         return super.renderTree(model);
     }
+
+    protected override onAfterShow(msg: Message): void {
+        super.onAfterShow(msg);
+        if (this.options.id === VSXExtensionsSourceOptions.INSTALLED) {
```

Comment:
This works when closing and re-opening the view. Unfortunately it seems that we do not have a notification system for when extensions are deployed, at least none that I can see and implementing one is probably out of scope here.

Reply:
> we do not have a notification system for when extensions are deployed

Yeah, I also found nothing there.
An other point I mentioned in the description is that, it would be nice to have a possibility to check if an extension is already installed. And what is even more important, to have a detailed return value when attempting to install an extension. Right now it is just a number of extensions being installed and if it is `0` an error is reported.

Reply:
I fully agree, there is definitely room for improvement on the exposure of extension management. I remember there being a discussion somewhere about being notified properly for uninstallations but I can't find it right now. Do you mind logging a Feature Request for the extension of that API?

Reply:
Sure, will do.